_data/compatibility/brd.yaml

```diff
@@ -0,0 +1,89 @@
+---
+name: BRD
+internal_url: /en/compatibility/brd
+logo: /img/compatibility/brd/brd.png
+rbf:
+  tested:
+    date: "2018-10-10"
+    platforms:
+      - iOS
+    version: "3.3.2"
+  features:
+    receive:
+      notification: "false"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "true"
+      shows_original_version: "true"
+    send:
+      signals_bip125: "false"
+      list: "untested"
+      details: "untested"
+      shows_replaced_version: "untested"
+      shows_original_version: "untested"
+  examples:
+    - image: /img/compatibility/brd/rbf/send-screen-default.png
+      caption: >
+        Sending Transaction - Transaction send default screen. No options for RBF here or in settings menus.
+    - image: /img/compatibility/brd/rbf/send-fee-options-expanded.png
+      caption: >
+        Sending Transaction - Transaction send screen with fee options expanded. No options for RBF. Toggle for fee priority.
+    - image: /img/compatibility/brd/rbf/send-transaction-confirmation.png
+      caption: >
+        Sending Transaction - Transaction send confirmation prompt.
+    - image: /img/compatibility/brd/rbf/transaction-details-sent.png
+      caption: >
+        Attempting Transaction Replacement - No ability to bump transaction fee found. Note Transaction not sent signaling RBF.
+    - image: /img/compatibility/brd/rbf/transaction-list-incoming-rbf.png
+      caption: >
+        Receiving Transaction Signaling RBF - No RBF flag on transaction list screen.
+    - image: /img/compatibility/brd/rbf/transaction-details-incoming-rbf.png
+      caption: >
+        Receiving Transaction Signaling RBF - No RBF flag on transaction details screen.
+    - image: /img/compatibility/brd/rbf/transaction-list-incoming-replacement.png
+      caption: >
+        Receiving Replacement Transaction - On transaction list screen, original transaction shows normal, replacement transaction shows with “Failed” message.
```

Comment:
I just want to confirm this caption is correct, it's the *replacement* that shows as "failed"?

Reply:
@harding yes, that is correct